www/templates/jsontestresults.html

```diff
@@ -1,17 +1,16 @@
 {% extends "test_results.html" %}
 
 {% block suites %}
-
+   <div class="content container">
     {% for suite in data.suites %}
         {% if suite.summary.result %}
             <div class="row">
-                <h2>Suite type: {{ suite.name }}</h2>
-                {% if data.utr_prefix %}
+                <h4>Suite type: {{ suite.name }}</h4>
+                {% if data.utrPrefix %}
```

Comment:
Is this the correct name for the variable name now? utrPrefix? I don't see where the change is on the data side